ofrak_core/test_ofrak/components/test_elf_load_aligment_modifier.py

```diff
@@ -96,6 +96,7 @@ async def verify_modifier_result(test_case: ElfModifierTestCase, output_path: st
         assert process.stdout == test_case.expect_stdout
 
 
+@pytest.mark.skipif_windows
```

Comment:
This test will also fail on macOS, but it will only be skipped on Windows, correct? 

Since this PR is Windows-specific, no need to do anything here. But probably worth making another issue documenting that these tests should only run on Linux.